ote/src/clj/ote/services/transport.clj

```diff
@@ -511,6 +530,22 @@
         (http/transit-response
           (business-id-exists db business-id)))
 
+      (POST "/transport-service/:service-id/associated-operators"
+        {{:keys [service-id]} :params
+         data :body}
+        (try
+          (http/transit-response
+            (save-associated-operator db service-id (http/transit-request data)))
+          (catch Exception e
+            {:status 403})))
+
+      (DELETE "/transport-service/:service-id/associated-operators/:operator-id"
+              {{:keys [service-id operator-id]} :params}
+        (if (= (delete-associated-operator db service-id operator-id) 1) ;Check if the amount of lines deleted is 1.
```

Comment:
Good that you added the requested comment, however if we're nitpicking a comment should explain _why_ something is done instad of _what_ (unless the line is exceptionally cryptic).

E.g. here something like "Successful deleting returns 1 changed rows, other responses shall be considered errors" or something better.

I guess this curren PR code will do, I know I'm nitpicking :)